instrumentation/spring/spring-jms/spring-jms-6.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/jms/v6_0/AbstractSpringJmsListenerTest.java

```diff
@@ -45,12 +45,12 @@ abstract class AbstractSpringJmsListenerTest {
   @BeforeAll
   static void setUp() {
     broker =
-        new GenericContainer<>("quay.io/artemiscloud/activemq-artemis-broker:artemis.2.27.0")
-            .withEnv("AMQ_USER", "test")
-            .withEnv("AMQ_PASSWORD", "test")
+        new GenericContainer<>("apache/activemq-artemis:2.44.0")
```

Comment:
i had issues running the other container, but this one worked well. If we'd prefer to keep it as is, I can revert

Reply:
this is much better to use the official image 👍